### ISSUE TITLE
Community guidelines

### ISSUE THREAD

**Issue by kylehamilton** [flor14/shinyssd#7]
This needs some community guidelines for people who wish to report issues or submit code to this project.

**Comment by flor14** [comment]
done

**Comment by karthik** [comment]
Agreed. This should be easy to do. If you install the `usethis` package, there is a function that can easily template this for you: `usethis::use_tidy_contributing()`

**Comment by flor14** [comment]
done
